interface/patient_file/encounter/forms.php

```diff
@@ -437,30 +427,30 @@ function divtoggle(spanid, divid) {
 
 <!-- Get the documents tagged to this encounter and display the links and notes as the tooltip -->
 <?php 
-	$docs_list = getDocumentsByEncounter($pid,$_SESSION['encounter']);
-	if(count($docs_list) > 0 ) {
+    $docs_list = getDocumentsByEncounter($pid,$_SESSION['encounter']);
+    if(count($docs_list) > 0 ) {
 ?>
 <div class='enc_docs'>
 <span class="bold"><?php echo xlt("Document(s)"); ?>:</span>
 <?php
-	$doc = new C_Document();
```

Comment:
This remark is a real need.  If we are going to flush out all tabs or something, then we should go ahead and get it done.  If we do that though, it needs to be done with a full-on "beautifier" that only uses spaces.  Removing double space before an operator for a single space and the like is just fine, but replacing a tab that functions as a single space with four or five spaces will result in some pretty ugly stuff.